docs/src/dev/provider/index.asciidoc

```diff
@@ -1007,56 +1167,41 @@ dependent on the operation being performed.  A `RequestMessage` has these fields
 [width="100%",cols="3,10",options="header"]
 |=========================================================
 |Key |Description
-|requestId |A link:http://en.wikipedia.org/wiki/Globally_unique_identifier[UUID] representing the unique identification for the request.
-|op |The name of the "operation" to execute based on the available `OpProcessor` configured in the Gremlin Server.  To evaluate a script, use `eval`.
-|processor |The name of the `OpProcessor` to utilize. The default `OpProcessor` for evaluating scripts is unnamed and therefore script evaluation purposes, this value can be an empty string.
-|args |A `Map` of arbitrary parameters to pass to Gremlin Server.  The requirements for the contents of this `Map` are dependent on the `op` selected.
+|gremlin |The script to be executed by the server.
+|language |The script's language.  The default is `gremlin-lang`.
+|timeoutMs |The maximum time a query is allowed to execute in milliseconds.  The default value is server dependent.
+|materializeProperties |Determines if properties will be included in `Element` results.
+|bindings |A `Map` of arbitrary parameters to pass to the server.  The requirements for the contents of this `Map` are dependent on the `op` selected.
+|g |The name of the `GraphTraversalSource` to use on the server.  The default is `g`.
 |=========================================================
 
-This message can be serialized in any fashion that is supported by Gremlin Server.  New serialization methods can
-be plugged in by implementing a `ServiceLoader` enabled `MessageSerializer`, however Gremlin Server provides for
+This message can be serialized in any fashion that is supported by Gremlin Server.  Gremlin Server provides for
 JSON serialization by default which will be good enough for purposes of most developers building drivers.
```

Comment:
> Gremlin Server provides for JSON serialization by default which will be good enough for purposes of most developers building drivers.

again, not sure we should be recommending JSON serialization drivers going forward, particularly for building them (probably not even a good statement in the old docs - probably a holdover from the Gryo days).